scripts/create-seed-fixture-heartwood

```diff
@@ -27,25 +31,90 @@ mkdir -p $TEST_REPO_PATH
 
 tar -xf $TEST_REPO_ARCHIVE -C $TEST_REPO_PATH
 
-RAD_HOME=$PALM_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad auth
+### ALICE ###
+
+export RAD_HOME=$ALICE_RAD_HOME
+export RAD_SEED=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
 
-GIT_AUTHOR_NAME="Alice Liddell"
-GIT_AUTHOR_EMAIL="alice@radicle.xyz"
-GIT_COMMITTER_NAME="Alice Liddell"
-GIT_COMMITTER_EMAIL="alice@radicle.xyz"
+export GIT_AUTHOR_NAME="Alice Liddell"
+export GIT_AUTHOR_EMAIL="alice@radicle.xyz"
+export GIT_COMMITTER_NAME="Alice Liddell"
+export GIT_COMMITTER_EMAIL="alice@radicle.xyz"
 
-RAD_SEED=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad auth
+eval $(ssh-agent)
+rad auth
+rad auth # first `rad auth` won't add key to ssh-agent
+radicle-node --listen 0.0.0.0:3444 &
 
 cd $ALICE_CHECKOUT
 
 git clone $TEST_REPO_PATH
 cd $TEST_REPO_NAME
+
+git checkout main
+rad init --name "source-browsing" \
+	--description "Git repository for source browsing tests" \
+	--default-branch "main" \
+	--no-confirm
+
 git checkout feature/branch
+git push rad
+
 git checkout orphaned-branch
-git checkout main
+git push rad
+
+### BOB ###
+
+export GIT_AUTHOR_NAME="Bob Belcher"
+export GIT_AUTHOR_EMAIL="bob@radicle.xyz"
+export GIT_COMMITTER_NAME="Bob Belcher"
+export GIT_COMMITTER_EMAIL="bob@radicle.xyz"
+export GIT_COMMITTER_DATE="Mon Nov 21 14:00 2022 +0100"
+
+export RAD_HOME=$BOB_RAD_HOME
+export RAD_SEED=fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe
+
+unset SSH_AUTH_SOCK
+unset SSH_AGENT_PID
+eval $(ssh-agent)
 
-RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad init --name "source-browsing" --description "Git repository for source browsing tests" --default-branch "main" --no-confirm
-PROJECT_ID=$(RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad .)
+rad auth
+rad auth
+radicle-node --listen 0.0.0.0:3445 \
```

Comment:
Could we change the setup slightly? It seems like now we connect the two peers first and then clone it onto the seed. The original setup was like this:

alice's peer creates a project
alice's peer publishes project onto seed
bob's peer fetches alice's project from the seed (not directly from alice)
bob's peer makes changes
bob's peer publishes changes to seed

Is it possible to do it this way with heartwood?

Reply:
[this](https://github.com/radicle-dev/radicle-interface/compare/c69755c740ef79f41bad616d8bd8413622703d0e..07db6d779f98a02970024d26f927517bf1342f82) is the best I can do atm